react/features/video-menu/components/web/RemoteVideoMenuTriggerButton.js

```diff
@@ -334,8 +353,30 @@ class RemoteVideoMenuTriggerButton extends Component<Props> {
         }
 
         if (buttons.length > 0) {
+            if (this.props._participantCount === 2) {
+                // This check was added because the issues was present
+                // in a two person meeting
+                if (this.props._filmstripHeight < 143) {
+                    return (
+                        <VideoMenu
+                            id = { participantID }>
+                            { buttons }
+                        </VideoMenu>
+                    );
+                }
+
+                return (
+                    <VideoMenu
+                        id = { participantID }
+                        style = { desktopVideoMenuStyles }>
+                        { buttons }
+                    </VideoMenu>
+                );
+            }
+
             return (
-                <VideoMenu id = { participantID }>
```

Comment:
Why do we need this change? I thought eslint allows placing a single prop on the same line as the component. Am I wrong?

Reply:
You are right. Updated it.